config.json

```diff
@@ -4,7 +4,7 @@
   "imgUrl": "logo.png",
   "execProgram": "node",
   "main": "app.js",
-  "version": "2.0.4",
+  "version": "2.1.0",
```

Comment:
please change the version to 3.0.0

Reply:
are you sure? This is from similar PR in EC2
<img width="936" alt="image" src="https://user-images.githubusercontent.com/92038478/157257384-7e7b7b32-1cd2-40cb-9228-1094294c735e.png">